skillImpacts/skillImpactsModel.js

```diff
@@ -0,0 +1,20 @@
+const db = require('../database/dbConfig');
+
+module.exports = {
+    findSkills
+};
+
+// SELECT * FROM skill_impact_requests JOIN project_goals
```

Comment:
no need to comment the actual SQL query here since the query builder underneath is self-explanatory

Reply:
This is just a query to help me visualize the function calls. Will remove that.